git.json

```
@@ -1222,6 +1225,39 @@
                 }
             ],
             "versions": "EQ(*)"
+        },
+        {
+            "action": "on poll",
+            "description": "Schedule regular cloning of a repository",
+            "verbose": "For regular cloning of a specified repository.",
+            "type": "ingest",
+            "identifier": "on_poll",
+            "read_only": false,
+            "parameters": {
+                "repo_url": {
+                    "description": "Repository URL",
+                    "verbose": "A repository URL that, if provided, will be cloned that is publicly accessible and does not require authentication.  If not provided, an asset must be configured for identifying which repository will be cloned",
+                    "data_type": "string",
+                    "primary": true,
+                    "contains": [
+                        "github repo"
+                    ],
+                    "order": 0
+                },
+                "branch": {
+                    "description": "Branch",
+                    "verbose": "A branch in the repository to clone, if required.  If not provided, this defaults to 'main', a branch that may or may not exist in the targeted repository",
+                    "data_type": "string",
+                    "primary": true,
+                    "contains": [
+                        "github branch"
```

This could be expanded
```suggestion
                        "github branch",
                        "gitlab branch",
                        "bitbucket branch",
                        "git branch"
```